src/QuickMarcEditor/QuickMarcEditWrapper.js

```diff
@@ -108,24 +116,30 @@ const QuickMarcEditWrapper = ({
       : searchParams.get('relatedRecordVersion');
 
     return mutator.quickMarcEditMarcRecord.PUT(marcRecord)
-      .then(() => {
+      .then(async () => {
         showCallout({
           messageId: marcType === MARC_TYPES.AUTHORITY
             ? 'ui-quick-marc.record.save.updated'
             : 'ui-quick-marc.record.save.success.processing',
         });
-        onClose();
+
+        refreshMarcRecord();
```

Comment:
There is also an issue that sometimes after saving a record - a close confirmation modal appears. This is happening because we're trying to redirect to url with new `relatedRecordVersion` before `initialValues` have updated. So we need to wait until all data is refreshed before redirecting:
1. In `QuickMarcEditorContainer` create `loadData` function. Let's copy everything from `useEffect` there. This function will be called inside the `useEffect` as before, and also passed as a prop `refreshPageData` to `Wrapper`.
2. Here instead of `refreshMarcRecord` call `await refreshPageData()`. 
```suggestion
        await refreshPageData();
        
        return { version: parseInt(instance._version, 10) + 1 };
```
Also it should be enough to just add +1 to previous version to not make too many requests

Reply:
I like it!  I was thinking about the cases if there was a change by another user. But yeah, it makes it a way simpler.

Reply:
We have the Optimistic locking feature to detect if we're trying to edit an outdated record. That's why we have the `relatedRecordVersion` in the url - if we're trying to edit record with version `5`, but someone else already made an edit and latest version is now `6` there'll be an optimistic locking banner
![image](https://user-images.githubusercontent.com/19309423/187881907-aeac7b2e-5446-4b46-8abc-443e7f50acc9.png)